pkg/config/template.go

```diff
@@ -345,6 +345,18 @@ allowed_annotations = [
 # Kata Containers with the Firecracker VMM
 #[crio.runtime.runtimes.kata-fc]
 
+#[crio.runtime.workloads.workload-type]
+# cpu_set = "0-1"
+# label = "workload-type.crio.io"
```

Comment:
nit: it'd be easier to check against if this was
`io.crio.workload-type` 
for instance the cpu one could be 
`io.crio.workload-type.cpus/container`, which would still allow us to check strings.Prefix(label) instead of having to check for a substring